src/test/moves/belly_drum.test.ts

```diff
@@ -0,0 +1,113 @@
+import {afterEach, beforeAll, beforeEach, describe, expect, test, vi} from "vitest";
+import Phaser from "phaser";
+import GameManager from "#app/test/utils/gameManager";
+import overrides from "#app/overrides";
+import {
+  TurnEndPhase,
+} from "#app/phases";
+import {getMovePosition} from "#app/test/utils/gameManagerUtils";
+import { Moves } from "#enums/moves";
+import { Species } from "#enums/species";
+import { BattleStat } from "#app/data/battle-stat";
+
+const TIMEOUT = 20 * 1000;
+const RATIO = 2;
+const PREDAMAGE = 15;
+
+describe("Moves - BELLY DRUM", () => {
+  let phaserGame: Phaser.Game;
+  let game: GameManager;
+
+  beforeAll(() => {
+    phaserGame = new Phaser.Game({
+      type: Phaser.HEADLESS,
+    });
+  });
+
+  afterEach(() => {
+    game.phaseInterceptor.restoreOg();
+  });
+
+  beforeEach(() => {
+    game = new GameManager(phaserGame);
+    vi.spyOn(overrides, "STARTER_SPECIES_OVERRIDE", "get").mockReturnValue(Species.MAGIKARP);
+    vi.spyOn(overrides, "OPP_SPECIES_OVERRIDE", "get").mockReturnValue(Species.SNORLAX);
+    vi.spyOn(overrides, "STARTING_LEVEL_OVERRIDE", "get").mockReturnValue(100);
+    vi.spyOn(overrides, "OPP_LEVEL_OVERRIDE", "get").mockReturnValue(100);
+    game.override.moveset([Moves.BELLY_DRUM]);
+    game.override.enemyMoveset([Moves.SPLASH]);
+  });
+
+  // Bulbapedia Reference: https://bulbapedia.bulbagarden.net/wiki/Belly_Drum_(move)
+
+  test("Belly Drum raises the user's Attack to its max, at the cost of 1/2 of its maximum HP",
+    async() => {
+      await game.startBattle([Species.MAGIKARP]);
+
+      const leadPokemon = game.scene.getPlayerPokemon();
+      expect(leadPokemon).toBeDefined();
+      const HpLost = Math.floor(leadPokemon.getMaxHp() / RATIO);
```

Comment:
nit: use camelCase
```suggestion
      const hpLost = Math.floor(leadPokemon.getMaxHp() / RATIO);
```
please apply to other occurences